GPU/Common/ReinterpretFramebuffer.cpp

```diff
@@ -0,0 +1,83 @@
+#include <cstdarg>
+
+#include "Common/GPU/Shader.h"
+#include "Common/GPU/ShaderWriter.h"
+#include "GPU/Common/ReinterpretFramebuffer.h"
+
+static const VaryingDef varyings[1] = {
+	{ "vec2", "v_texcoord", "TEXCOORD0" },
+};
+
+// TODO: We could have an option to preserve any extra color precision. But gonna start without it.
+// Requires full size integer math.
```

Comment:
you could double the widths of the channels (10-12-10 instead of 5-6-5 etc.), although not sure if it is worth it ...

Reply:
Yeah let's think more about this. I feel that it should be possible somehow but...